lib/features.ts

```diff
@@ -95,3 +96,12 @@ export function isIOS () {
     // @ts-expect-error - MSStream is not in `window` type, unsurprisingly
     (/iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream);
 }
+
+export function isMobileSafari18 () {
+  if (isBrowser()) {
+    const { browser, os } = new UAParser().getResult();
+    return os.name?.toLowerCase() === 'ios' && !!browser.name?.toLowerCase()?.includes('safari')
+      && browser.major === '18';
```

Comment:
Should this be changed to cover @louiecampagna-okta's comment?
```suggestion
    const { browser } = new UAParser().getResult();
    return !!browser.name?.toLowerCase()?.includes('safari')
      && browser.major === '18';
```

Reply:
nvm. just saw your reply. we'll update separately after more troubleshooting info
@louiecampagna-okta

Reply:
Good suggestion. This would also help with desktop mode in Safari on iOS